### DISCUSSION TITLE
Orama Search not wroking in nextra - 3.0.0-alpha.24

### DISCUSSION THREAD

**Discussion by Shubham-EV** [oramasearch/orama#745] [General]
I am using,
    "nextra": "^3.0.0-alpha.24",
    "nextra-theme-docs": "^3.0.0-alpha.24",
    
The official [orama nextra plugin](https://docs.orama.com/open-source/plugins/plugin-nextra) is not working.

**Comment by lazarusA** [comment]
Indeed, not sure if there are missing instructions:

For me, doing the following a new project doesn't  take care of required dependencies. Is this expected, or are we missing information in the docs? https://docs.orama.com/open-source/plugins/plugin-nextra

```sh
npm install @orama/plugin-nextra
```

```
npm install @orama/plugin-nextra
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @orama/plugin-match-highlight@3.1.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```
